roiextractors/extractors/numpyextractors/numpyextractors.py

```diff
@@ -131,24 +138,46 @@ def __init__(self, image_masks, signal,
         if isinstance(image_masks, (str, Path)):
             image_masks = Path(image_masks)
             signal = Path(signal)
+            dff = Path(signal) if dff is not None else Path('none.npy')
```

Comment:
this will fail when loading the `none.py` as it doesn't exist. I would check later:

if dff.is_file():
      load